src/Features/Core/Portable/Navigation/NavigationOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Navigation
+{
+    internal static class NavigationOptions
+    {
+        public const string FeatureName = "Navigation";
+
+        public static readonly Option<bool> UsePreviewTab = new Option<bool>(FeatureName, "UsePreviewTab", defaultValue: false);
```

Comment:
Yes, I'll add a comment. I'll also rename it to something a bit clearer, like PreferProvisionalTab or something like that.